### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by GautamSharda**
Test pull request

### CODE REVIEW

**Comment by lucac123** [comment]
idk it looks fine

File: AmongUs.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from discord.ext import commands
+from discord.utils import get
+
+
+client = commands.Bot(command_prefix = '.')
+
+@client.event
+async def on_ready():
+    print('Among Us bot is ready!')
+
```
Comment (lucac123) [inline AmongUs.py:11]: what is this blank line doing here
waste of space

**Comment by GautamSharda** [comment]
yah man it's king shit

File: AmongUs.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import discord
```
Comment (lucac123) [inline AmongUs.py:1]: 2/5

File: AmongUs.py (hunk 1 above)
Comment (GautamSharda) [reply AmongUs.py:11]: lul sry master i like da aesthetic is it wrong

**Review by lucac123 — APPROVED** [review]
idk wut this is

**Pull request merged**